planning/obstacle_stop_planner/src/node.cpp

```diff
@@ -99,6 +99,8 @@ ObstacleStopPlannerNode::ObstacleStopPlannerNode(const rclcpp::NodeOptions & nod
     p.unknown_lateral_margin =
       declare_parameter<double>(ns + "detection_area.unknown_lateral_margin");
     p.extend_distance = declare_parameter<double>(ns + "detection_area.extend_distance");
+    p.enable_stop_behind_goal_for_obstacle =
+      declare_parameter<bool>(ns + "detection_area.enable_stop_behind_goal_for_obstacle", true);
```

Comment:
@ismetatabay please remove default parameter
```suggestion
      declare_parameter<bool>(ns + "detection_area.enable_stop_behind_goal_for_obstacle");
```

Reply:
done :+1: